setup_windows.py

```diff
@@ -3,21 +3,63 @@
 from distutils.msvccompiler import get_build_version
 
 
+def get_default_connector(client):
+    if client == "mariadbclient":
+        return os.path.join(
+            os.environ["ProgramFiles"], "MariaDB", "MariaDB Connector C"
+        )
+    elif client == "mysqlclient":
+        return os.path.join(
+            os.environ["ProgramFiles"], "MySQL", "MySQL Connector C 6.1"
```

Comment:
Don't! Don't use too old version by default.

Reply:
This is only the default path if someone explicitly set the client type to "mysqlclient". The default client type is "mariadbclient". Also, this default connector path is the same as is currently the default in site.cfg. Yes, this version of the MySQL Connector/C is very old, but to my knowledge there is no newer version. I believe they stopped publishing a C Connector. This is why I suggested dropping support for the MySQL C Connector entirely. If that is acceptable to you, I will add another commit to this PR that removes support for MySQL Connector/C and only uses MariaDB Connector/C.

Reply:
FWI, they provide mysqlclient.lib in MySQL server for now.

Reply:
Ah ok. Would you prefer then to continue supporting mysqlclient and I can see if I can find a default MySQL Server install directory for it?